shared/common-adapters/avatar.js.flow

```diff
@@ -0,0 +1,41 @@
+// @flow
+import * as React from 'react'
+import type {StylesCrossPlatform, StylesCrossPlatformWithSomeDisallowed} from '../styles'
+
+export type AvatarSize = 128 | 96 | 64 | 48 | 32 | 16
+
+type DisallowedStyles = {
+  borderStyle?: empty,
+}
+
+export type Props = {
```

Comment:
It's pretty confusing to have `Props` here actually be `OwnProps` in `avatar.js`. Can you make the names match, i.e. rename `Props` here to `OwnProps`, and have another `Props` type that `mockOwnToViewProps` below returns?

Reply:
avatar is connected so inside itself it has OwnProps to signify the props that come externally. From the outside those are actually Props. If we rename it, it should be Props instead. But honestly how this is connected is very messy, if we want to fix this up and remove this facade we should just do that